src/OpenGLES/EAGLContext.cs

```diff
@@ -15,6 +20,29 @@ public EAGLContext ()
 		{
 		}
 #endif
+
+		[iOS (10,0)]
+		[TV (10,0)]
+		public bool PresentRenderBuffer (nuint target, double presentationTime)
```

Comment:
check `api-diff` - in most case what we move from generated to manual bindings needs to be `virtual` (and removing `virtual` is a breaking change)